openslides/motions/models.py

```diff
@@ -464,11 +460,25 @@ def reset_state(self, workflow=None):
 
     def get_agenda_title(self):
         """
-        Return a title for the agenda.
+        Return a simple title string for the agenda.
+
+        Contains only agenda item number and title.
```

Comment:
This `Contains` is missleading because this method does not return an agenda item number but only the motion title. Of course the agenda code adds the agenda item number later but is can't be seen here.
Proposal: `Returns only the motion title so that you have only agenda item number and title in the agenda.`